www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx

```diff
@@ -24,187 +24,113 @@ import {
+  toDateTime: (date: string | Date) => string;
 }
 
 const getDetailCardLines = ({
   details,
-  toDate,
-  toTime,
-}: DetailCardLineProps): Array<DetailCardLines> => {
-  const getDateTimeLines = ({ label, field }): DetailCardLines => ({
-    title: label,
-    field,
-    getLines: (): Lines => [
-      {
-        key: `${label}_date`,
-        line: <DetailsLine line={toDate(field)} />,
-      },
-      {
-        key: `${label}_time`,
-        line: <DetailsLine key="tries" line={toTime(field)} />,
-      },
-    ],
-  });
-
-  const getCheckLines = ({ label, field }): DetailCardLines => ({
-    ...getDateTimeLines({ label, field }),
-    getLines: (): Lines => [
-      ...getDateTimeLines({ label, field }).getLines(),
-      {
-        key: `${label}_active`,
-        line: details.active_checks ? <ActiveLine /> : null,
-      },
-    ],
-  });
-
+  toDateTime,
+}: DetailCardLineProps): Array<DetailCardLine> => {
   return [
     {
       title: labelFqdn,
       field: details.fqdn,
       xs: 12,
-      getLines: (): Lines => [
-        {
-          key: 'fqdn',
-          line: <DetailsLine line={details.fqdn} />,
-        },
-      ],
+      line: <DetailsLine line={details.fqdn} />,
     },
     {
       title: labelAlias,
       field: details.alias,
-      getLines: (): Lines => [
-        {
-          key: 'fqdn',
-          line: <DetailsLine line={details.alias} />,
-        },
-      ],
+      line: <DetailsLine line={details.alias} />,
     },
     {
       title: labelMonitoringServer,
       field: details.monitoring_server_name,
-      getLines: (): Lines => [
-        {
-          key: 'poller',
-          line: <DetailsLine line={details.monitoring_server_name} />,
-        },
-      ],
+      line: <DetailsLine line={details.monitoring_server_name} />,
     },
     {
       title: labelTimezone,
       field: details.timezone,
-      getLines: (): Lines => [
-        {
-          key: 'timezone',
-          line: <DetailsLine line={details.timezone} />,
-        },
-      ],
+      line: <DetailsLine line={details.timezone} />,
     },
     {
       title: labelCurrentStateDuration,
       field: details.duration,
-      getLines: (): Lines => [
-        { key: 'duration', line: <DetailsLine line={details.duration} /> },
-        {
-          key: 'tries',
-          line: <DetailsLine key="tries" line={details.tries} />,
-        },
-      ],
+      line: <DetailsLine line={`${details.duration} - ${details.tries}`} />,
     },
-    getDateTimeLines({
-      label: labelLastStateChange,
+    {
+      title: labelLastStateChange,
       field: details.last_status_change,
-    }),
-    getCheckLines({ label: labelLastCheck, field: details.last_check }),
-    getCheckLines({ label: labelNextCheck, field: details.next_check }),
+      line: <DetailsLine line={toDateTime(details.last_status_change)} />,
+    },
+    {
+      title: labelLastCheck,
+      field: details.last_check,
+      active: details.active_checks,
```

Comment:
not related to this PR, but this information is a bit wrong
it should be useful to get a tile which indicated if the service is actively and/or passively checks
But, we can't know if the last or next checks ar passive or active